examples/projects/dagster-slurm-example/dagster_slurm_example/resources/__init__.py

```diff
@@ -26,9 +26,9 @@
 
 RESOURCES_PROD = {"pipes_subprocess_client": pipes_subprocess_client}
 resource_defs_by_deployment_name = {
-    "dev": RESOURCES_LOCAL,
-    "staging": RESOURCES_STAGING,
-    "prod": RESOURCES_PROD,
+    "dev": RESOURCES_LOCAL,  ## fully local package on fly
```

Comment:
SSH/Slurm Integration: Missing proper resource abstraction for SSH connection management